packages/atomic-react/package.json

```diff
@@ -21,7 +21,18 @@
     "build:assets": "ncp ../atomic/dist/atomic/assets dist/assets && ncp ../atomic/dist/atomic/lang dist/lang "
   },
   "main": "./dist/cjs/index.js",
-  "module": "./dist/index.esm.js",
+  "exports": {
```

Comment:
Would these be helpful (for now or future)
https://www.npmjs.com/package/package-json-exports
https://www.npmjs.com/package/eslint-import-resolver-exports

Reply:
- https://www.npmjs.com/package/package-json-exports

While I understand the motivation, I don't think it'd be necessarily a good fit or idea.
Indeed, this is an easy way to end up with a lot of exports and entrypoints, which imo has adverse consequences for every party involved:
 - For the maintainer, they have more entrypoints when doing changes, and also more chance to mess up.
 - For the user, they may need to have lines and lines of import, or have 'choices' to make "should I import X from subModuleA or subModuleB??". This could lead to an unpleasant UX I think.
 
 Right now, gut feeling, we're going toward having entrypoints per 'use-cases'. Those entrypoints are to be manually crafted.
 
 - https://www.npmjs.com/package/eslint-import-resolver-exports
Yes this one (and the one it refer) would be useful to ensure we don't break an entrypoint when moving files around.
Nice one :D